spec/features/idv/doc_auth/how_to_verify_spec.rb

```diff
@@ -72,4 +72,135 @@
       expect(page).to have_current_path(idv_hybrid_handoff_url)
     end
   end
+
+  describe 'navigating to How To Verify from Agreement page in 50/50 state' do
+    context 'opt in false at start but true during navigation' do
```

Comment:
in_person_proofing_opt_in_enabled can be enabled and a user can opt in or opt out. I think we should be very specific here. What do you think about something like 'in_person_proofing_opt_in_enabled is false at start but true during navigation' rather than 'opt in false'? If you like this, I'd suggest we make this change for each context block.